src/components/importerTab/LoadDataSubmenu.tsx

```diff
@@ -110,10 +112,10 @@ export function LoadDataSubmenu() {
       <Flex style={{ minHeight: 100 }}>
         <Flex vertical gap={10} style={{ display: currentStage >= 1 ? 'flex' : 'none' }}>
           <Text>
-            {`File contains ${currentSave.relics.length} relics and ${currentSave.characters.length} characters. Replace your current data with the uploaded data?`}
+            {t('stage2.label', { reliccount: currentSave.relics.length, charactercount: currentSave.characters.length })}
           </Text>
           <Button style={{ width: importerTabButtonWidth }} icon={<ImportOutlined/>} type='primary' onClick={loadConfirmed} loading={loading2}>
```

Comment:
Do you think its worth keeping a commented version of translated strings in the code in English for readability? It looks really hard to navigate the codebase now. Something like: 

```
{t('stage2.label', { reliccount: currentSave.relics.length, charactercount: currentSave.characters.length })}
// Translation: {`File contains ${currentSave.relics.length} relics and ${currentSave.characters.length} characters. Replace your current data with the uploaded data?`}
```

Reply:
yeah that's probably not a bad idea